easybuild/easyconfigs/m/Mesa/Mesa-10.4.5-intel-2015a-Python-2.7.9.eb

```diff
@@ -0,0 +1,67 @@
+easyblock = 'ConfigureMake'
+
+name = 'Mesa'
+version = '10.4.5'
```

Comment:
How does this relate to #1226?

I put a lot of effort in avoid all `osdependencies` there, but got stuck because of `libudev`...

I'd like to see a Mesa easyconfig without OS deps, and I think we're quite close with #1226...